### DISCUSSION TITLE
SectorComputeProofFailed: not on curve

### DISCUSSION THREAD

**Discussion by aaron3481** [filecoin-project/lotus#12800] [Storage Provider]
Hi all,

I am running 1.31.0 and I build from source with SupraSeal C2 enabled. When I try to pledge a sector, after PC1 and PC2, log shows SectorComputProof error: not on curve

I search on Google but can find any help. Any idea?

**Comment by rjan90** [comment]
Hey @aaron3481! 

Do you have the full error output? Most `not on curve` errors stem from and is caused by compute/memory errors - so could be worth doing memory checks. Ref: https://github.com/filecoin-project/lotus/issues/2051

**Reply by beck-8** [reply]
mem or gpu

**Comment by rjan90** [comment]
Hey @aaron3481! Just following up to hear if you where able to do a memory check or switch out the GPU to get error go away?
